ci-operator/config/red-hat-data-services/ods-ci/red-hat-data-services-ods-ci-master__rhoai2.13-ocp4.18-lp-interop.yaml

```diff
@@ -0,0 +1,68 @@
+base_images:
+  cli:
+    name: "4.18"
+    namespace: ocp
+    tag: cli
+  rhoai-tests:
+    name: ods-ci
+    namespace: ci
+    tag: dev
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.18"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: rhoai-interop-aws
+  cron: 0 0 11 6 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
```

Comment:
We should use another `BASE_DOMAIN` for our tests.
@liswang89 Lets discuss how should we create it in our sync

Reply:
There is `rhods.ccitredhat.com` public to use in RHOAI, see [AWS Route 53 record](https://us-east-1.console.aws.amazon.com/route53/v2/hostedzones?region=us-west-2#ListRecordSets/Z06924683R75LHMN6FU5H)